common/dynamicconfig/constants.go

```diff
@@ -761,6 +761,11 @@ This config is EXPERIMENTAL and may be changed or removed in a later release.`,
 		0.5,
 		`HistoryHostErrorPercentage is the percentage of hosts that are unhealthy`,
 	)
+	HistoryHostStartingPercentage = NewGlobalFloatSetting(
+		"frontend.historyHostStartingPercentage",
+		0.05,
+		`HistoryHostStartingPercentage is the percentage of hosts that are not yet ready to serve traffic`,
```

Comment:
Maybe say a little more here about what happens when the percentage is above or below this threshold?

Reply:
I think this is up to the consumer of the float setting, not the float setting itself.

Reply:
But the setting has a single meaning and a specific effect.. the docstring should describe those. And the name should too, ideally. There shouldn't be multiple consumers with different semantics for the same setting.

Anyway, 0.05 isn't "the percentage of hosts that are not yet ready to serve traffic", the percentage of hosts not ready to serve traffic is usually 0%, sometimes 10%, sometimes 50%, etc. The setting is a threshold for some behavior to change based on that value.

Reply:
I see your point. I've changed the dynamic config documentation and the naming to better reflect its intent.